index.js

```diff
@@ -8,19 +8,15 @@ module.exports = {
     ecmaVersion: 2018
   },
   plugins: ['mocha'],
-  extends: 'eslint:recommended',
+  extends: ['eslint:recommended', 'plugin:prettier/recommended'],
```

Comment:
This might cause a bunch of linting errors if anyone updates... Just something to think/talk about.

Reply:
That's true, we could just add `prettier` to the list of plugins instead. Would be good to see if or how this changes linting for existing repos using both eslint and prettier.

Reply:
I would imagine it would lead to some pretty heavy code changes. It would be good to allow typescript linting without making prettier a requirement. 

We could potentially just add it to the extensions of the typescript override at the very least. 

What would adding it to the plugins alone do?

Reply:
That might be better, I don't think this repo actually had prettier integrated with it very well... by adding it as a plugin alone, any declared prettier rules that generate warnings would be forwaded to and handled by eslint. See [this repo](https://github.com/prettier/eslint-plugin-prettier)

Reply:
got it. Unfortunately I think that would also make it harder to gradually update to TS for current repos, which might slow down adoption efforts. Maybe we could add prettier as a separate release after the typescript release? Just to give people the option?